frontend/ui/network/networklistener.lua

```diff
@@ -16,6 +16,10 @@ local NetworkListener = EventListener:extend{
     _activity_check_delay_seconds = nil,
 }
 
+if not Device:hasWifiToggle() then
+    return NetworkListener
+end
```

Comment:
Because I checked as I wasn't sure: nothing else requires NetworkListener, so by the time FM/RD do, Device is guaranteed to have been initialized, so this works perfectly fine ;).